### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by jlvaillant** [jlvaillant/intellicenter#43] [General]
- compatibility update for HA 2022.2.6
- compatibility update for Pentair firmware 1.064
- stability improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/jlvaillant/intellicenter/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by TheRydad** [comment]
Thanks for updating! I am no longer seeing the ping/pong timeout problem. This is awesome!

**Comment by roopesh** [comment]
Thank you!
